package.json

```diff
@@ -89,6 +93,7 @@
     "lerna": "^2.1.2",
     "link-parent-bin": "^0.2.0",
     "markdown-magic": "^0.1.25",
+    "puppeteer": "^1.11.0",
```

Comment:
This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.